staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -65,3 +65,17 @@ func (o *Options) Apply() {
 func (o *Options) Get() (logr.Logger, error) {
 	return logRegistry.Get(o.LogFormat)
 }
+
+// ValidateLogFormat validates LogFormat field
+func ValidateLogFormat(logFormat string) error {
```

Comment:
Like mentioned in other comment. I don't think this code will be used beside Kubelet. Let's inline it